modules/distribution/src/assembly/bin.xml

```diff
@@ -540,6 +546,44 @@
             </source>
             <outputDirectory>wso2is-${pom.version}/repository/conf/identity</outputDirectory>
         </file>
+	<!--file>
+		    <source>../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/bps.xml
+		    </source>
+		    <outputDirectory>wso2is-${pom.version}/repository/conf/</outputDirectory>
+        	<fileMode>644</fileMode>
+        </file-->
+	    <file>
+		    <source>../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/database/jpadb.h2.db
+		    </source>
+		    <outputDirectory>wso2is-${pom.version}/repository/database/</outputDirectory>
+        	<fileMode>644</fileMode>
+        </file>
+	    <file>
+		    <source>../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/datasources/bps-datasources.xml
+		    </source>
+		    <outputDirectory>wso2is-${pom.version}/repository/conf/datasources/</outputDirectory>
+        	<fileMode>644</fileMode>
+        </file>
+		<file>
+		    <source>../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/b4p-coordination-config.xml
+		    </source>
+		    <outputDirectory>wso2is-${pom.version}/repository/conf/</outputDirectory>
+        	<fileMode>644</fileMode>
+        </file>
+
+		<file>
+		    <source>../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/humantask.xml
+		    </source>
+		    <outputDirectory>wso2is-${pom.version}/repository/conf/</outputDirectory>
+        	<fileMode>644</fileMode>
+        </file>
+
+       <file>
+            <source>
+            	src/repository/resources/bps/bps.xml
+	    </source>
+            <outputDirectory>wso2is-${pom.version}/repository/conf/</outputDirectory>
+        </file>
 
```

Comment:
Can we have the bps.xml in distribution/conf folder? This is the usual place we have duplicated config files if necessary.